apps/web/src/components/docs/docs.const.ts

```diff
@@ -2,6 +2,7 @@ import { ROUTES } from '../../constants/routes';
 
 export const PATHS: { [key in ROUTES]?: string } = {
   [ROUTES.ACTIVITIES]: 'activity-feed/introduction',
+  // TODO: Where is this endpoint after IA?
   [`${ROUTES.BRAND}/layouts`]: 'content-creation-design/layouts',
```

Comment:
@antonjoel82 @BiswaViraj is `/brand/layout` page still around post IA? My understanding is that it's not but I want to double-check with you.

Reply:
This should be removable now as the mapping on the row below it should be the new router for layouts.